pages/_app.tsx

```diff
@@ -1,15 +1,31 @@
-import '../styles/globals.css';
-import type { AppProps } from 'next/app';
-import { Provider } from "react-redux";
-import { store } from "../redux/store";
-import { ThemeProvider } from '@space-metaverse-ag/space-ui';
-
-function App({ Component, pageProps }: AppProps) {
-  return <Provider store={store}>
+import { Provider } from 'react-redux'
+
+import { ThemeProvider, GlobalStyles } from '@space-metaverse-ag/space-ui'
+import TopNav from 'layouts/TopNav'
```

Comment:
issue with this import, casing is wrong: 
![image](https://user-images.githubusercontent.com/27584221/198678483-85153481-3664-49e5-be3d-ec5775e3c27e.png)

Reply:
For me this error does not appear, but I made an update in the eslint settings to fix it

Reply:
hm, I still have error, I updated the casing.